code/__DEFINES/misc.dm

```diff
@@ -141,7 +141,7 @@
 //2 = reduced hud (just hands and intent switcher)
 //3 = no hud (for screenshots)
 
-#define MINERAL_MATERIAL_AMOUNT 2000
+#define MINERAL_MATERIAL_AMOUNT 1000
```

Comment:
Hahahahaha, fixed? I was the one who fixed it, and believe me, there is still a ton of hardcoded amounts.

And you are wrong, base materials (m, g) were 3750 and the rest were 2000. Then the base ones were changed to 2000 for consistency.